<?xml version="1.0" ?>
<sdf version="1.4">
  <model name="camera_support">
    <static>0</static>
    <link name="link">

      <inertial>
        <mass>0.5</mass>
        <inertia>
          <ixx>0.000015</ixx>
          <ixy>0.0</ixy>
          <ixz>0.0</ixz>
          <iyy>0.000015</iyy>
          <iyz>0.0</iyz>
          <izz>0.0000108</izz>
        </inertia>
      </inertial>

      <collision name="collision1">
        <pose>0 0 0.0 0 0 0</pose>
          <geometry>
            <box>
              <size>0.15 0.15 0.005</size>
            </box>
          </geometry>
      </collision>
      <collision name="collision2">
          <pose>0 0 0.255 0 0 0</pose>
          <geometry>
            <cylinder>
              <radius>0.012</radius>
              <length>0.5</length>
            </cylinder>
          </geometry>
      </collision>

      <visual name="visual">
        <pose>0 0 -0.002 0 0 0</pose>
        <geometry>
          <mesh>
             <uri>model://camera_support/meshes/camera_support.dae</uri>
    		  </mesh>
        </geometry>
      </visual>

      <velocity_decay>
        <linear>0.000000</linear>
        <angular>0.000000</angular>
      </velocity_decay>
      <self_collide>0</self_collide>
      <kinematic>0</kinematic>
      <gravity>1</gravity>
    </link>
  </model>
</sdf>
